docs/dev/developer-guides/transactions/fee-model.md

```diff
@@ -1,72 +1,104 @@
+
```

Comment:
high level comment:

this document is covering both very high level concepts (like 'refunds') with very detailed information (how we replace the signatures when doing fee estimations) - and they are 'combined' together.

I'd suggest putting the high level information on top of the doc and then the details.

things that should be on top:
* that gas consists of 2 parts (compute + publishing to L1)
* that we do refunds - and that some tooling might not show it correctly

And only then diving into details.

Reply:
Disagree as this doc will get bigger, no doubt, and what appears now to be just high level information may have more detailed content in the future. Ideally, we would have one page per topic in lower level menus.